sdks/java/core/src/main/java/org/apache/beam/sdk/metrics/LabeledMetricNameUtils.java

```diff
@@ -63,8 +65,12 @@ public void addLabel(String key, String value) {
           .append(LABEL_DELIMITER);
     }
 
+    public void addMetricLabel(String key, String value) {
```

Comment:
needs a comment, it is a little confusing on how this is different from addLabel